diff --git a/NetRocks/src/Op/OpXfer.cpp b/NetRocks/src/Op/OpXfer.cpp
index 583b7508..68949f96 100644
--- a/NetRocks/src/Op/OpXfer.cpp
+++ b/NetRocks/src/Op/OpXfer.cpp
@@ -350,6 +350,11 @@ void OpXfer::Transfer()
 						? XOA_OVERWRITE : XOA_ASK;
 				}
 
+				std::string prefix = "/tmp/far2l";
+				if (e.first.size() >= prefix.size() && e.first.compare(0, prefix.size(), prefix) == 0) {
+					xoa = XOA_OVERWRITE;
+				}
+
 				if (xoa == XOA_ASK) {
 					xoa = ConfirmOverwrite(_kind, _direction, path_dst, e.second.modification_time, e.second.size,
 								existing_file_info.modification_time, existing_file_info.size).Ask(_default_xoa);
